tests/containers/host_configuration.pm

```diff
@@ -21,16 +21,19 @@ use version_utils 'check_os_release';
 sub run {
     my ($self) = @_;
     $self->select_serial_terminal;
+    my $interface = script_output qq(ip r s default | awk '{print \$5}' | head -1);
     if (check_os_release('suse', 'PRETTY_NAME')) {
+        validate_script_output qq(ip a s '$interface'), sub { m/((\d{1,3}\.){3}\d{1,3}\/\d{1,2})/ };
         ensure_ca_certificates_suse_installed();
     }
     else {
         assert_script_run "dhclient -v";
+        validate_script_outputqq(ip a s '$interface'), sub { m/((\d{1,3}\.){3}\d{1,3}\/\d{1,2})/ };
```

Comment:
I would also move this line up. And maybe also the `dhclient`..

Reply:
`dhclient` is only for the non-SUSE case and the `ip a` is to verify that an IP got assigned, so has to come after `dhclient`